v0/destinations/ProfitWell/transform.js

```diff
@@ -0,0 +1,186 @@
+const get = require("get-value");
+const logger = require("../../../logger");
+const { EventType } = require("../../../constants");
+const { getSubscriptionHistory } = require("./nethandler");
+const {
+  CustomError,
+  getDestinationExternalID,
+  defaultRequestConfig,
+  defaultPostRequestConfig,
+  defaultPutRequestConfig,
+  removeUndefinedAndNullValues,
+  constructPayload,
+  getSuccessRespEvents,
+  getErrorRespEvents,
+  getFieldValueFromMessage
+} = require("../../util");
+const { ConfigCategory, mappingConfig, baseEndpoint } = require("./config");
+
+const identifyResponseBuilder = async (message, { Config }) => {
+  const response = defaultRequestConfig();
+
+  const userId = getDestinationExternalID(message, "profitwellUserId");
+  const userAlias = getFieldValueFromMessage(message, "userId");
+
+  if (!userId && !userAlias) {
+    throw new CustomError("User not found", 400);
+  }
+
+  let subscriptionId = getDestinationExternalID(
+    message,
+    "profitwellSubscriptionId"
+  );
+  let subscriptionAlias =
```

Comment:
lets make subscriptionId or subscriptionAlias required from our side. As in atleast one of them is required. Otherwise, there is no point in making the `get` call as well